biomodels/README.md

```diff
@@ -0,0 +1,62 @@
+
+This tool will insert the cross-references for the BioModels Reference Database into the appropriate pathway instances of the release database.
+
+It will:
+
+ - Read the models2pathways.tsv file describing the relationships between Reactome pathways and BioModels identifiers.
+ - Create a BioModels Reference Database instance (if one does not already exist)
+ - Create the Database Identifier instances representing the BioModels identifiers and insert them into Pathway instances under the 'crossReference' attribute
+ 
+ ```
+## models2pathways.tsv example
+BIOMD0000000498 R-HSA-917977    8.191346134722721E-7    https://reactome.org/PathwayBrowser/#/R-HSA-917977      Transferrin endocytosis and recycling   IEA     Homo sapiens
+BIOMD0000000287 R-HSA-69563     6.066213418698396E-6    https://reactome.org/PathwayBrowser/#/R-HSA-69563       p53-Dependent G1 DNA Damage Response    IEA     Homo sapiens
+BIOMD0000000287 R-HSA-2559586   6.066213418698396E-6    https://reactome.org/PathwayBrowser/#/R-HSA-2559586     DNA Damage/Telomere Stress Induced Senescence   IEA     Homo sapiens
+```
+
+For the purposes of this module, the first two columns are the most important. The first column contains BioModels identifiers, and the second column contains Reactome Pathway identifiers that have have been connected to them by biomodels-mapper. For each line in `models2pathways.tsv`, the corresponding Pathway instance in Reactome's relationship database will receive a CrossReference that points to the biomodel identifier. If the biomodel identifier does not already have a database instance, it will be created before being slotted into the Pathway instance as an instance of type `DatabaseIdentifier`. All biomodels identifier instances are connected to the sole 'BioModels Database' `ReferenceDatabase` object. 
+ 
+## Logging
+ 
+This application will log to a file under `./logs/biomodels-{timestamp}.log` and `./logs/biomodels-{timestamp}.err` for warnings and errors. This file will have the same content as the console (stdout/stderr) except without the log4j prefix. 
+ 
+## Configuration
+
+The configuration file is set up like a typical properties file, containing database adaptor information and the database ID of your person instance:
+
+```
+### Sample configuration file with properties
+release.database.host=mySQLHost
+release.database.user=mySQLUsername
+release.database.password=mySQLPassword
+release.database.port=3306
+release_current.name=release_current
+personId=123456
+```
+Once this has been completed and saved somewhere on the server, you should be able to run the biomodels insertion step.
+  
+## Compiling & Running
+
+This is a Java application which requries a Java 8+ environment. You will need maven and a full JDK to compile.
+
+To compile the application, run this command:
+
+```
+$ mvn clean compile package
```

Comment:
It should be enough to run `mvn clean package`, I think.  Since the `package` life-cycle phase occurs after the compile phase, it should first do a compile followed by a package.

Reply:
After removing the EBI dependencies and installing release-common-lib locally (with the most up to date version that is seen on develop), it only produces a jar file without the dependencies included. I'm not entirely sure why, maybe this is all we need or I'm missing something? Maybe we can look at it tomorrow if you have time

Reply:
That being said, there was no difference in `mvn clean package` and `mvn clean compile package`, so this is still correct

Reply:
Rhis might be a target that needs ‘mvn clean compile assembly:single’

Reply:
Yes that worked